.github/workflows/build.yml

```diff
@@ -116,3 +116,35 @@ jobs:
     - run: make clang-release-ubsan
     - run: make test-clang-debug-ubsan
     - run: make test-clang-release-ubsan
+
+  build-min-cmake:
+    name: min-cmake
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        submodules: true
+    - uses: actions/setup-python@v1
+      with:
+        python-version: '3.x'
+    - name: Install Ninja
+      run: sudo apt-get install ninja-build
+    - name: Detect minimum CMake version
+      run: >
+        awk 'match($0, /cmake_minimum_required\(VERSION *([0-9]+\.[0-9]+)\)/, a)
+        { print "WABT_CMAKE_VER=" a[1] }' CMakeLists.txt | tee $GITHUB_ENV
+    - name: Install minimum CMake
+      run: |
+        python -m pip install -U setuptools wheel pip
+        python -m pip install "cmake==${WABT_CMAKE_VER}.*"
+        cmake --version
+    - name: Configure WABT
+      run: cmake -G Ninja -S . -B out -DCMAKE_BUILD_TYPE=Release
```

Comment:
How do we know that the cmake that pip installs will end up first in the PATH? Could we add a check on the output of `cmake --version` and have it bomb out if it doesn't match (or, install that version to a particular location and then execute cmake with a fully qualified pathname)?

Reply:
I'll check the former. I'm not sure the path is stable.

Reply:
Done